src/lib/structures/Transformer.ts

```diff
@@ -1,23 +1,33 @@
 import { TransformerInvalidFormatterError } from '../errors/TransformerInvalidFormatterError';
+import { Unreachable } from '../errors/Unreachable';
 import type { ISentencePart } from './ISentencePart';
 
 export class Transformer implements ISentencePart {
-	private readonly kRun: IFormatter;
+	public readonly name: string;
 
 	public constructor(name: string) {
-		const formatter = Transformer.kFormatters.get(name);
+		const formatter = Transformer.formatters.get(name);
 		if (typeof formatter === 'undefined') {
 			throw new TransformerInvalidFormatterError(this, name);
 		}
 
-		this.kRun = formatter;
+		this.name = name;
+		this.run = formatter;
 	}
 
-	public run(value: string): string {
-		return this.kRun(value);
+	public run(value: string): string;
+	public run(): string {
```

Comment:
```suggestion
	public run(value?: string): string {
```

No need for an overloaded function. I'm shocked ESLint didn't even complain about this. I guess that rule isn't enabled.

Reply:
It is needed, because the variable `value` is unused.

Also, it's not an overload, but a signature.